src/components/Person/Person.jsx

```diff
@@ -1 +1,47 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const SEX_FEMALE = 'f';
+const SEX_MALE = 'm';
+
+function getPartnerInformation(sex, partnerName) {
+  if (sex === SEX_FEMALE) {
+    return (
+      <p className="Person__partner">{`${partnerName} is my husband`}</p>
+    );
+  }
+
+  if (sex === SEX_MALE) {
+    return (
+      <p className="Person__partner">{`${partnerName} is my wife`}</p>
+    );
+  }
```

Comment:
Maybe we can simplify this. The only difference between these 2 returns is the partner `wife` or `husband`. So better to use ternary here